src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -332,69 +157,125 @@ spinner(void *d)
 		;
 }
 
+void
+clear_sched(int* rcvd, thdid_t* tid, int* blocked, cycles_t* cycles, tcap_time_t* thd_timeout)
```

Comment:
looks like you don't use any of the output parameters here, perhaps just don't have them at all. Coz, "clear" is just saying, just remove and ignore all sched events. 
Name of the function be more documentable, "sched_events_clear" or "sched_queue_clear" or something.

Reply:
Agreed